docs/physics/montecarlo/index.rst

```diff
@@ -4,11 +4,11 @@
 Monte Carlo Iteration
 *********************
 
-After setting up the simulation, TARDIS runs the simulation using the ``.run()`` method. This runs several Monte
-Carlo iterations (which will be described in the links below), corresponding to the number of iterations specified
+After setting up the simulation, TARDIS runs the simulation using the ``.run_convergence()`` method. This runs several Monte
+Carlo iterations (which will be described in the links below), corresponding to one less than the number of iterations specified
```

Comment:
This suggests to me that we should consider the "final" run to *always* occur while the number of iterative runs are optional and user-defined.